src/main/java/seedu/sudohr/logic/commands/employee/DeleteCommand.java

```diff
@@ -46,6 +46,8 @@ public CommandResult execute(Model model) throws CommandException {
         model.deleteEmployee(employeeToDelete);
         model.cascadeDeleteEmployeeToDepartments(employeeToDelete);
         model.cascadeDeleteUserInLeaves(employeeToDelete);
+        model.updateFilteredDepartmentList(Model.PREDICATE_SHOW_ALL_DEPARTMENTS);
```

Comment:
actually, can i double check, if i delete an employee from dept, will the count on ur end be correctly updated as well?